common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1304,22 +1305,22 @@
     "figura.docs.text_task.set_alignment": "Sets this Text alignment\nCan be either \"LEFT\", \"RIGHT\" or \"CENTER\"\nDefault \"LEFT\"",
     "figura.docs.text_task.has_shadow": "Checks if this task text has shadow",
     "figura.docs.text_task.set_shadow": "Toggles if the Text should render with a drop shadow\nNot compatible with \"Outline\" mode",
-    "figura.docs.text_task.has_outline": "Checks if this task text has outline",
-    "figura.docs.text_task.set_outline": "Toggles if the Text should render with a outline\nAlways renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
+    "figura.docs.text_task.has_outline": "Checks if this task text has an outline",
+    "figura.docs.text_task.set_outline": "Toggles, if the text should render with an outline\nAlways, renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
     "figura.docs.text_task.get_outline_color": "Gets this tasks text outline color",
     "figura.docs.text_task.set_outline_color": "Sets the outline color this Text should render\nOnly compatible with \"Outline\" mode",
     "figura.docs.text_task.get_width": "Gets this Text max width\nDefault 0",
-    "figura.docs.text_task.set_width": "Sets this Text max width, wrapping the text into multiple lines\nWidth of 0 or less do not wraps the text\nDefault 0",
+    "figura.docs.text_task.set_width": "Sets this Text max width, wrapping the text into multiple lines\nWidth of 0 or less do not wrap the text\nDefault 0",
     "figura.docs.text_task.has_wrap": "Check if this Text should wrap lines",
     "figura.docs.text_task.set_wrap": "Sets if this Text should wrap lines",
     "figura.docs.text_task.is_see_through": "Check if this Text can be seen behind walls\nDefault false",
     "figura.docs.text_task.set_see_through": "Sets if this Text can be seen behind walls\nDefault false",
     "figura.docs.text_task.has_background": "Check if this Text should render its background\nDefault false",
     "figura.docs.text_task.set_background": "Sets if this Text should render its background\nDefault false",
-    "figura.docs.text_task.get_background_color": "Gets the this Text background color",
-    "figura.docs.text_task.set_background_color": "Sets the background colour of this Text\nIf the alpha value is not given, it will uses the vanilla value (as in the accessibility settings)",
-    "figura.docs.text_task.get_opacity": "Gets the opacity of this text",
-    "figura.docs.text_task.set_opacity": "Sets the opacity of this text",
+    "figura.docs.text_task.get_background_color": "Gets this Text's background color",
+    "figura.docs.text_task.set_background_color": "Sets the background color of this text\nIf the alpha value is not given, it will use the vanilla value (as in the accessibility settings)",
+    "figura.docs.text_task.get_opacity": "Gets the opacity of this Text",
+    "figura.docs.text_task.set_opacity": "Sets the opacity of this Text",
```

This time text being capitalized seems deliberate. Does "Text" here refer to the task itself or was this done for consistency with the other strings? I'm not sure if it would still make sense to capitalize it.
```suggestion
    "figura.docs.text_task.get_background_color": "Gets this text's background color",
    "figura.docs.text_task.set_background_color": "Sets the background color of this text\nIf the alpha value is not given, it will use the vanilla value (as in the accessibility settings)",
    "figura.docs.text_task.get_opacity": "Gets the opacity of this text",
    "figura.docs.text_task.set_opacity": "Sets the opacity of this text",
```

---

I changed it for consistency but I think you're right.